e2b.Dockerfile

```diff
@@ -12,3 +12,7 @@ RUN curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | s
 	&& echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
 	&& sudo apt update \
 	&& sudo apt install gh -y
+
+# Install Python
+RUN sudo apt update \
```

Comment:
python is already installed in the template. Actually python should be in ubuntu by default

Reply:
if i remember correctly "python3" worked... but will test again